File: src/Libraries/HealthInsureSystem.Entities/Concrete/Insured.cs
using HealthInsureSystem.Entities.Abstract;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HealthInsureSystem.Entities.Concrete
{
    public class Insured : IEntity
    {
        public int Id { get; set; }
        public string InsuredFirstName { get; set; }
        public string InsuredLastName { get; set; }    
        public string IdentityNumber { get; set; }
        public int BirthYear { get; set; }
        public string Gender { get; set; }
        public string Degree { get; set; }
        public decimal Weight { get; set; }
        public decimal Height { get; set; }
        public decimal BodyMassIndex { get; set; }
        public bool Status { get; set; }
        public int CustomerId { get; set; }
        public int PolicyId { get; set; }
    }
}

File: src/Presentation/HealthInsureApp/src/app/models/insureds/insured.ts
export interface Insured {
  id: number;
  insuredFirstName: string;
  insuredLastName: string;
  identityNumber: string;
  birthYear: number;
  gender: string;
  degree: string;
  weight: number;
  height: number;
  bodyMassIndex: number;
  status: boolean;
  customerId: number;
  policyId: number;
}
